external/Makefile

```diff
@@ -124,7 +139,7 @@ endif
 endif
 
 custom_clean:
-	$(foreach OBJFILE, $(OBJS), rm -f $(OBJFILE))
+	$(foreach OBJFILE, $(OBJS) $(LIBBUNDLED) $(LIBEXTERNAL), rm -f $(OBJFILE))
 
 clean: custom_clean iotivity_clean
 	$(call DELFILE, $(BIN))
```

Comment:
The commit message and the change donot match. Commit message says linking the .o and .a. But in the modification you are deleting the files.  Can you reframe your commit message to convey the correct functionality